### PR TITLE
Added in TS1/2 Beam Current stats above the graph in big letters

### PR DESCRIPTION
**Pull request by SophieKSTFC**
### Description of work

I have added in the Beam Current for both TS1 and TS2 in big letters above the graph.

### Ticket

[#4297](https://github.com/ISISComputingGroup/IBEX/issues/4297)

### Acceptance criteria

- [x] The TS1 and TS2 beam currents are clearly displayed above the grave when viewing the 'Beam Status' view on the GUI.
- [x] The beam currents update as expected and are correct compared to the information provided in the graph and Beam Information Panel

### Documentation

*Highlight and provide a link to any additions or changes to the documentation, if applicable. The aim is provide information to help the reviewer*

---

#### Code Review

- [x] Is the code of an acceptable quality?
- [x] Do the changes function as described and is it robust?
- [x] Have the changes been documented in the [release notes](https://github.com/ISISComputingGroup/IBEX/wiki/ReleaseNotes_Dev)?

### Final Steps
- [ ] Reviewer has moved the [release notes](https://github.com/ISISComputingGroup/IBEX/wiki/ReleaseNotes_Dev) entry for this ticket in the "Changes merged into master" section

### CODE REVIEW

**Comment by Tom-Willemsen** [comment]
- The UI is a bit confusing at the moment - it looks like the radio buttons apply to the new labels rather than the graph as they are in the same grouping container. How about moving the radio buttons to the bottom to make it clear they apply to the graph only?
- Can we format the numbers to a consistent number of decimal places - currently it fluctuates between displaying 2 and 3 d.p. which looks a bit odd.

**Comment by SophieKSTFC** [comment]
On investigation, seems that the muon target uA is not a PV we easily know how to read.

**Comment by SophieKSTFC** [comment]
Have moved the time period buttons below the beam currents and formatted the values to 2 decimal places.

File: base/uk.ac.stfc.isis.ibex.beamstatus/src/uk/ac/stfc/isis/ibex/beamstatus/internal/NumberConverter.java [hunk 1]
```diff
@@ -33,7 +33,6 @@ public String convert(Number value) throws ConversionException {
 		if (value == null) {
 			return "UNKNOWN";
 		}
-		
-		return value.toString();
+		return String.format("%.02f", value);//value.toString();
```
Comment (adamb56789) [inline base/uk.ac.stfc.isis.ibex.beamstatus/src/uk/ac/stfc/isis/ibex/beamstatus/internal/NumberConverter.java]: Remove comment

File: base/uk.ac.stfc.isis.ibex.ui.beamstatus/src/uk/ac/stfc/isis/ibex/ui/beamstatus/views/BeamGraphView.java [hunk 2]
```diff
@@ -23,6 +23,8 @@
 import java.util.Arrays;
 import java.util.Optional;
 
+import java.text.DecimalFormat;
```
Comment (adamb56789) [inline base/uk.ac.stfc.isis.ibex.ui.beamstatus/src/uk/ac/stfc/isis/ibex/ui/beamstatus/views/BeamGraphView.java]: Remove unused import

File: base/uk.ac.stfc.isis.ibex.ui.beamstatus/src/uk/ac/stfc/isis/ibex/ui/beamstatus/views/BeamGraphView.java [hunk 3]
```diff
@@ -55,231 +67,330 @@
...
+		createTSCurrentLabel(rowOne, 2);
+		createButton(rowTwo, "Last 24 Hours", MILLISECONDS_IN_DAY);
+		createButton(rowTwo, "Last Hour", MILLISECONDS_IN_HOUR);
+
+	}
+	
+	/*
+	 * Creates a Composite with a row layout.
+	 */
+	private Composite createCompositeRow(final Composite parent){
```
Comment (adamb56789) [inline base/uk.ac.stfc.isis.ibex.ui.beamstatus/src/uk/ac/stfc/isis/ibex/ui/beamstatus/views/BeamGraphView.java]: Checkstyle warning (spacing)

File: base/uk.ac.stfc.isis.ibex.ui.beamstatus/src/uk/ac/stfc/isis/ibex/ui/beamstatus/views/BeamGraphView.java [hunk 4]
```diff
@@ -55,231 +67,330 @@
...
+		
+		Composite controlsComposite = new Composite(parent, SWT.NONE);
+		controlsComposite.setLayoutData(new GridData(SWT.CENTER, SWT.TOP, false, false));
+		controlsComposite.setSize(SWT.FILL, 20);
```
Comment (adamb56789) [inline base/uk.ac.stfc.isis.ibex.ui.beamstatus/src/uk/ac/stfc/isis/ibex/ui/beamstatus/views/BeamGraphView.java]: For all of the magic numbers, suppress the warning

**Comment by SophieKSTFC** [comment]
fixed check style warnings, removed unused code and imports. Fixed ibex gui build errors.

**Comment by SophieKSTFC** [comment]
have renamed the checkstyle config name to comply with IBEX standard

**Pull request merged**